From 321aeff15042ece8ff6f3804c8cec1dc41185485 Mon Sep 17 00:00:00 2001
From: Flora Cui <Flora.Cui@amd.com>
Date: Sun, 9 Oct 2016 16:32:23 +0800
Subject: [PATCH 1583/2094] drm/amd/amdgpu: add new vram flag for dgma

add AMDGPU_GEM_CREATE_VRAM_CONTIGUOUS for dgma reserved bo

Change-Id: Id0e2a3f171644a5f1257dad9f08f8d22ae0996ed
Signed-off-by: Flora Cui <Flora.Cui@amd.com>
Reviewed-by: Hawking Zhang <Hawking.Zhang@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c
index 6b68367..25ca06f 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c
@@ -1140,7 +1140,8 @@ static int amdgpu_direct_gma_init(struct amdgpu_device *adev)
 	size = (unsigned long)amdgpu_direct_gma_size << 20;
 	/* reserve in visible vram */
 	r = amdgpu_bo_create(adev, size, PAGE_SIZE, true, AMDGPU_GEM_DOMAIN_VRAM,
-			     AMDGPU_GEM_CREATE_CPU_ACCESS_REQUIRED | AMDGPU_GEM_CREATE_TOP_DOWN,
+			     AMDGPU_GEM_CREATE_CPU_ACCESS_REQUIRED | AMDGPU_GEM_CREATE_VRAM_CONTIGUOUS |
+			     AMDGPU_GEM_CREATE_TOP_DOWN,
 			     NULL, NULL, &abo);
 	if (unlikely(r))
 		goto error_out;
-- 
2.7.4

